block/manager.go

```diff
@@ -1165,4 +1162,13 @@ func updateState(s *types.State, res *abci.ResponseInitChain) {
 	// We update the last results hash with the empty hash, to conform with RFC-6962.
 	s.LastResultsHash = merkle.HashFromByteSlices(nil)
 
+	if len(res.Validators) > 0 {
+		vals, err := cmtypes.PB2TM.ValidatorUpdates(res.Validators)
+		if err != nil {
+			panic(err)
```

Comment:
Why are we panicing here instead of simply returning an error?

Reply:
because the `updateState` func doesn't return error,

Reply:
Well the `updateState` function didn't return an error previously because there were no error conditions. Now there are so we should return an error.